Assets/AnkrSDK/Examples/UseCases/WebGlLogin/WebGLLoginPanelController.cs

```diff
@@ -0,0 +1,77 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using AnkrSDK.Data;
+using AnkrSDK.Utils;
+using TMPro;
+using UnityEngine;
+using UnityEngine.Events;
+
+namespace AnkrSDK.UseCases.WebGlLogin
+{
+	public class WebGLLoginPanelController : MonoBehaviour
+	{
+		[Serializable]
+		public class PanelEventWithWallet : UnityEvent<WebGL.SupportedWallets>{}
+		
+		[Serializable]
+		public class PanelEventWithNetwork : UnityEvent<NetworkName>{}
```

Comment:
why do you use unity events? This does not help to read the code, it is impossible to understand without launching the editor and it is also prone to errors while merging scenes and prefabs.

Reply:
Got it. In what cases do you recommend to use UnityEvent.

Reply:
Never use it:)

Reply:
It is a convenient tool for visual programming but this is not the case for us.